docs-source/content/_index.md

```diff
@@ -91,6 +91,10 @@ Documentation for APIs:
 
 * See the [Javadoc](https://oracle.github.io/weblogic-kubernetes-operator/apidocs/index.html) for the operator.
 
+### Oracle support
+
+To access Oracle support for running WebLogic Server domains on Kubernetes platforms, see [WebLogic Server Certifications on Kubernetes](https://support.oracle.com/epmos/faces/DocumentDisplay?_afrLoop=208317433106215&id=2349228.1&_afrWindowMode=0&_adf.ctrl-state=c2nhai8p3_4).
```

Comment:
Can we make reference to My Oracle Support Doc ID 2349228.1

Reply:
Sure, so you would prefer that it say, see "My Oracle Support Doc ID 2349228.1" along with the hyperlink, instead of "WebLogic Server Certifications on Kubernetes"? Is that what you are proposing?

Reply:
see [WebLogic Server Certifications on Kubernetes My Oracle Support Doc ID 2349228.1]

Reply:
ok, I get it; will do.